paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/Settings.kt

```diff
@@ -27,12 +27,9 @@ class Settings(context: Context) {
     internal companion object {
         /**
          * Note: only necessary if not configured via `gradle.properties`.
-         *
-         * Set to the base URL of your test backend. If you are using
-         * [example-mobile-backend](https://github.com/stripe/example-mobile-backend),
-         * the URL will be something like `https://hidden-beach-12345.herokuapp.com/`.
+         * Set to the base URL of your test backend.
          */
-        private const val BASE_URL = "put your base url here"
+        private const val BASE_URL = "https://stripe-mobile-payment-sheet.glitch.me/"
```

Comment:
could you add some explanation about the url here in the javadoc?

Reply:
Done